modules/test/playwright/tests/object-web/objectEntries.spec.ts

```diff
@@ -867,6 +869,7 @@ test.describe('Manage object entries through Workflow', () => {
 				{code: 0},
 				undefined,
 				'default',
+				undefined,
```

Comment:
Hey @cgmonte, just curious. Is this change just in the name of good practice or did we have any problems with these requests after the API changes made by headless?

Reply:
We need to pass undefined because the arguments are ordered (because of headless' changes) and we don't want to pass some of the arguments. I was passing null before the latest push, and in this case, null does not work the same. Scope was being defined as null and random post of object definitions were failing.